### PR TITLE
chore: add github templates for issue and pr

### LINKED ISSUE #6: create issue and pull request templates
as title

### PR DESCRIPTION
**Pull request by zhu-xiaowei**
1. add bug_report and feature_report template for issue.
2. add pr template.

Close #6

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```
Comment (zxkane) [inline .github/ISSUE_TEMPLATE/config.yaml:1]: prefer to change it as `false`, always using templates to create issue

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,6 +21,64 @@ reported the issue. Please try to include as much information as you can. Detail
...
+    - For a bugfix: `fix: description of changes`
+    - For a feature: `feat: add awesome feature`
+    - Everything else: `chore: fix build script`
+- No period at the end of the title.
+- Pull Request message should indicate which issues are fixed: `fixes #<issue>` or `closes #<issue>`.
+- If not obvious (i.e. from unit tests), describe how you verified that your change works.
+- If this PR includes breaking changes, they must be listed at the top of the changelog as described above in the Pull Request Checklist.
+- PR must be reviewed by at least one repository maintainer, in order
+  to be considered for inclusion.
+- PR must also pass the CodeBuild workflow like Checkstyle, Lint, and unit tests.
```
Comment (zxkane) [inline CONTRIBUTING.md:77]: we don't use CodeBuild here

**Review by zxkane — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yaml (hunk 1 above)
Comment (zhu-xiaowei) [reply .github/ISSUE_TEMPLATE/config.yaml]: changed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (zhu-xiaowei) [reply CONTRIBUTING.md]: deleted.

**Pull request merged**